Description: Do not use embedded jsencrypt
 Use stuff from libjs-jsencrypt
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-08-15

--- python-xstatic-jsencrypt-2.0.0.2.orig/xstatic/pkg/jsencrypt/__init__.py
+++ python-xstatic-jsencrypt-2.0.0.2/xstatic/pkg/jsencrypt/__init__.py
@@ -34,9 +34,9 @@ HOMEPAGE = 'http://travistidwell.com/jse
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/jsencrypt'
+BASE_DIR = '/usr/share/javascript/jsencrypt'
 
 LOCATIONS = {
     # CDN locations (if no public CDN exists, use an empty dict)
